javascript/javascript2/week1/recipe-form.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+   <title>Create Your Own Recipe</title>
+    <link rel="stylesheet" href="recipe-form-style.css" />
+  </head>
+  <body>
+    <header>
+      <div id="home-page">Recipe</div>
+      <div>Create new Recipe</div>
+    </header>
+    <main>
+      <h1>Create your own Recipe below</h1>
+      <form action="">
+        <div class="recipe-name"><label for="recipe-name">Recipe name: </label>
+        <input
+          type="text"
+          id="recipe-name"
+          placeholder="Please enter recipe name"
+        /></div>
+        
+<div class="recipe-img"><label for="">Image URL:</label>
```

Comment:
You've missed to add the `for` attribute
```suggestion
     <div class="recipe-img">
        <label for="recipe-image">Image URL:</label>
```